changelog/core-memory-__delete.dd

```diff
@@ -30,6 +30,6 @@ $(CONSOLE
 sed "s/delete \(.*\);/__delete(\1);/" -i **/*.d
 )
 
-Users should prefer $(REF destroy, object)` to explicitly finalize objects,
-and only resort to $(REF __delete, core,memory) when $(REF destroy, object)
+Users should prefer $(REF1 destroy, object)` to explicitly finalize objects,
```

Comment:
There's a dangling backtick here.

Reply:
Fixed (as the changelog file has been removed now.)